x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -935,6 +940,7 @@ const processRecoveredAlerts = ({
     const baseContext = {
       alertDetailsUrl: getAlertDetailsUrl(basePath, spaceId, alertUuid),
       group: hasGroupBy(validatedParams) ? recoveredAlertId : null,
+      groupByKeys: groupByKeysObjectForRecovered[recoveredAlertId],
```

Comment:
Just out of curiosity: Do we have a test to check what is generated in the context?

Reply:
I don't believe we have tests to check context of `recovered` alerts. I will add this in a separate PR.